src/Controller/UI/DatasetReviewController.php

```diff
@@ -479,6 +479,7 @@ public function postAction(Request $request, int $id = null, MessageBusInterface
                 foreach ($fileset->getNewFiles() as $file) {
                     $file->setStatus(File::FILE_IN_QUEUE);
                 }
+                $datasetSubmission->setDatasetFileTransferStatus(DatasetSubmission::TRANSFER_STATUS_BEING_PROCESSED);
```

Comment:
If we are going to set it, it needs to be outside the fileset condition because it is not dependent on that